ballerina/tests/tests.bal

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import paypal.orders.mock as _;
+
+import ballerina/os;
+import ballerina/test;
+import ballerina/uuid;
+
+configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
+
+configurable string clientId = os:getEnv("PAYPAL_CLIENT_ID");
+configurable string clientSecret = os:getEnv("PAYPAL_CLIENT_SECRET");
+
+final string serviceUrl = isLiveServer ? "https://api-m.sandbox.paypal.com/v2/checkout" : "http://localhost:9090";
+final string tokenUrl = isLiveServer ? "https://api-m.sandbox.paypal.com/v1/oauth2/token" : "http://localhost:9444/oauth2/token";
+
+Client paypal = test:mock(Client);
+
+string captureOrderId = "";
+string captureOrderPaymentCaptureId = "";
+string captureOrderTrackingId = "";
+
+string authorizeOrderId = "";
+
+purchase_unit_request[] purchaseUnits = [
+    {
+        amount: {
+            value: "200.00",
+            currency_code: "USD",
+            breakdown: {
+                item_total: {
+                    currency_code: "USD",
+                    value: "180.00"
+                },
+                shipping: {
+                    value: "20.00",
+                    currency_code: "USD"
+                }
+            }
+        }
+    }
+];
+
+@test:BeforeSuite
+function initClient() returns error? {
+    if (isLiveServer) {

```

Comment:
```suggestion
    if isLiveServer {
```

Refer to the Best Practices in Ballerina: https://learn-ballerina.github.io/index.html